### PR TITLE
Add EIP: Gas Fee Sponsorship

### PR DESCRIPTION
**Pull request by Toni-d-e-v**
**ATTENTION: ERC-RELATED PULL REQUESTS NOW OCCUR IN [ETHEREUM/ERCS](https://github.com/ethereum/ercs)**

--

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-7745.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+eip: 7745
```
Comment (abcoathup) [inline EIPS/eip-7745.md:2]:
```suggestion
eip: 7735
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: EIPS/eip-7745.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+title: Gas Fee Sponsorship
+description: Enable one Ethereum address to sponsor gas fees for another by authorizing and co-signing transactions.
+author: Toni Dumančić (@toni-d-e-v)
+discussions-to: https://ethereum-magicians.org/t/eip-7745/7765
```
Comment (abcoathup) [inline EIPS/eip-7745.md:6]: Please create an actual topic in Eth Magicians with a link to this PR.

Comment (Toni-d-e-v) [reply EIPS/eip-7745.md:6]: I don't have an option to create anything on the website 😥

Comment (abcoathup) [reply EIPS/eip-7745.md]: There should be a "New Topic" button on the right hand side on Eth Magicians. 
image

If this doesn't show, you may need to spend some time on the site first reading other posts.

Comment (Toni-d-e-v) [reply EIPS/eip-7745.md]: Yeah, could you just create it and send me the link.

**Comment by lightclient** [comment]
duplicate of #2711

**Pull request closed without merging**